webui/react/src/pages/ResourcepoolDetail.tsx

```diff
@@ -197,4 +200,10 @@ const ResourcepoolDetail: React.FC = () => {
   );
 };
 
-export default ResourcepoolDetail;
+const ResourcepoolDetailPage: React.FC = () => (
+  <Suspense fallback={<Page loading />}>
```

Comment:
- in this case, I think `<Spinner />` would be more suitable
- Also, I couldnt see loading animation in this `Suspense`

Reply:
Updated. I notice now that there's actually a skeleton for this page while it's loading. Can anybody help me figure out where that happens? That's where the suspense should live but I couldn't find the code that triggers that.

Reply:
Okay, I finally had a chance to look into this. So the reason the suspense here is never hit is because of timing around different API calls but it's not _inherently_ inaccessible. Once we convert more data management to use this new pattern we should see the suspense being hit.